### ISSUE TITLE
[Issue?] Tidy 5e Compatibility 

### ISSUE THREAD

**Issue by Foundry37** [ironmonk88/monks-enhanced-journal#207]
![image](https://user-images.githubusercontent.com/99896703/154752482-fe4776dd-665b-40f9-93ce-b44f5dda4e02.png)

I've been having trouble with "Tidy5e Sheet" in the Dark alternate mode, which we've always used for character sheets.

When you swap over, obviously the text changes over from black on a grey-cream background to bright white and shades of grey on the black background to be more visible, yet weirdly when Tidy5e Sheet Dark is in effect actors in journal entries only get half the changeover? The dark background comes in, but the text remains black and basically invisible.

No idea what's going on with that and haven't been able to find a setting mishap that's causing it.

**Comment by etriebe** [comment]
Yeah, seems like Tidy5e sheet is completely broken when an actor is linked from within Monk's Enhanced Journal.

**Comment by webmaster94** [comment]
This does appear to be somewhat mitigated by the "Open Links" option but the embedded entry is still broken.

**Comment by MrZwij** [comment]
I have the same issue with the Tidy 5e sheet, where most of the NPC actions, features, etc. are missing (as seen in the image above). First reported on Discord here: https://discord.com/channels/862414864692281386/862421185211924540/1004205322493251644

![image](https://user-images.githubusercontent.com/12764695/182524031-db6a5c12-05f8-4d22-9f1d-4b450dcc05a0.png)

**Comment by bularzik** [comment]
I tried to reproduce this on Foundry v14 build 14.365 with the current dnd5e system and couldn't — this went through a follow-up repro pass with the Tidy5e Sheet module installed and wasn't reproducible. This might have been fixed as a side effect of other changes, or may be specific to an older Foundry/system version. If you're still seeing this on a current build, could you confirm your Foundry/MEJ/system versions and any console errors?
